src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/api/UpsertAllocationResponse.kt

```diff
@@ -14,4 +14,8 @@ data class UpsertAllocationResponse(
   @Schema(description = "Whether or not the allocation was created", required = true)
   @NotNull
   val created: Boolean,
+
+  @Schema(description = "Prison code", required = true)
+  @NotNull
+  val prison: String,
```

Comment:
are we in control of this schema - seems a bit naff that it isn't `prisonId` - `prison` on its own could be the name, not the coe